src/graphql/schema.gql

```diff
@@ -222,6 +222,8 @@ type Space {
   validation: Strategy
   followersCount: Int
   proposalsCount: Int
+  parent: Space
```

Comment:
Did you gave up on using String?

Reply:
I planned to inform you before marking this ready for review. ;) Gonna add some explanation. Open for discussion and changes. But so far I think it makes sense like this.